### PR TITLE
Setting up the base for React application using Create React App

### LINKED ISSUE #774: Start off a React application for Bassa with the help of Webpack and Babel
**Is your feature request related to a problem? Please describe.**

To create a React application for Bassa, a boilerplate code for a React application is needed to initiate the workflow of the project.

**Describe the solution you'd like**
I would like to create an independent React application in the root folder of Bassa (not in /ui, but outside of that) and create a simple working React application.

**Describe alternatives you've considered**
I have considered two approaches here. With the discussion of my mentors, I will pick one of them.
First one is to use Create-React-App command for building React application
Pro(s): it is well-equipped with all the modules that should be required for building a React application. From modules requiring for development to testing, all the factors have been taken care of.
Con(s): Drawback is that number of modules that come pre-installed make the application heavy and slow. It would take more time to load Javascript to the HTML page.

Other one which I think is building React application without using Create-React-App command.

Pro(s): We can use minimal set of modules required to develop a React application. Of course, webpack will be used for bundling and Babel will be used for transpiling JSX to browser-compatible Javascript.

Con(s): We might face some issues and need to add some extra modules while adding more features to the application like file-loaders, image-loaders and other loaders and plugins to successfully run the application. It may take some time to identify the correct set of loaders and plugins to use in the project.

### PR DESCRIPTION
**Pull request by singhrohan62**
Setting up the base for React application using Create React App command. This is the first pull request for Project: `Shifting Bassa codebase from Angular to React`

## Description
Right now, I have just initiated a React application with Create React App, in the next commit for the same PR, I will be adding Material-UI in the application and later a new PR for dashboard component.

## Related Issue
Fixes #774 and #777 

## How Has This Been Tested?
It has been tested locally on my PC.
To test the application, type in terminal:
*`cd components/frontend-react`
*`sudo npm start`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/frontend-react/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (agentmilindu) [inline components/frontend-react/public/manifest.json]: Let's change these accordingly

Comment (singhrohan62) [reply components/frontend-react/public/manifest.json]: I will be updating it accordingly.

**Comment by singhrohan62** [comment]
@agentmilindu I made changes in manifest file and I checked on my phone screens, it's working. Also, I replaced default React icons with that of Bassa icons. Also, I made small change in App component.

**Comment by agentmilindu** [comment]
@vivonk @charithccmc Can you please review?

**Pull request merged**